veniq/dataset_collection/augmentation.py

```diff
@@ -577,8 +619,8 @@ def save_text_to_new_file(input_dir: Path, text: str, filename: Path) -> Path:
                 if (iteration_number % iteration_cycle) == 0:
                     df.to_csv(csv_output)
                 iteration_number += 1
-            except Exception:
-                continue
+            except Exception as e:
+                print(str(e))
```

Comment:
```suggestion
                print(e)
```

Reply:
it's doesn't; work with JavaLang exceptions. only str(e) works